tools/generate_icons.dart

```diff
@@ -0,0 +1,59 @@
+import 'dart:convert';
+import 'dart:io';
+
+/// Phosphor Icons generator
+/// reads the phosphor json and generates a dart class
+/// with all the phosphor icons constants
+void main(List<String> arguments) {
+  final jsonFile = File(arguments.first);
+  assert(!jsonFile.existsSync());
+
+  final icons = json.decode(jsonFile.readAsStringSync())['icons'];
+
+  final fileLines = [
+    """// Auto Generated File
+import 'package:phosphor_flutter/src/phosphor_icon_data.dart';
+
+abstract class PhosphorIcons {""",
+  ];
+  final constantsLines = <String>[];
+
+  icons.forEach((icon) {
```

Comment:
Ditto here, let's fail if all 5 weights are not present on each icon.

Reply:
ditto, check for missing styles and used the new regex